pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -626,3 +632,12 @@ func (r *ReconcileComplianceSuite) getAffectedMcfgPool(scan *compv1alpha1.Compli
 func remediationNeedsOutdatedRemoval(rem *compv1alpha1.ComplianceRemediation, suite *compv1alpha1.ComplianceSuite) bool {
 	return suite.ShouldRemoveOutdated() && rem.Status.ApplicationState == compv1alpha1.RemediationOutdated
 }
+
+func (r *ReconcileComplianceSuite) setSuiteMetric(suite *compv1alpha1.ComplianceSuite) error {
+	if suite.Status.Result == compv1alpha1.ResultCompliant {
+		r.metrics.SetComplianceStateInCompliance(suite.Name)
+	} else if suite.Status.Result == compv1alpha1.ResultNonCompliant {
+		r.metrics.SetComplianceStateOutOfCompliance(suite.Name)
+	}
```

Comment:
let's also handle the `INCONSISTENT` state (as a `2` maybe); this is also quite worrying to find in clusters and should be alerted on.

Reply:
thanks, done.